### DISCUSSION TITLE
Initial map options

### DISCUSSION THREAD

**Discussion by AdamSnow** [Freika/dawarich#121] [Ideas]
Wondering about options for the initial load of the map. I find that with heatmap showing, it really seems like my server is working hard to show that data on load. I think it'd be great if I could set some initial map settings... Default to "Last 7 days" with no heatmap just lines for example.

**Comment by Freika** [comment]
@AdamSnow this is a good idea! I'll move map options to settings, so each user could decide what they want to be enabled by default in one of the future releases!
